### PR TITLE
Adds ability to disseminate custom high-priority alerts in Puget Sound

### PR DESCRIPTION
**Pull request by aaronbrethorst**
* Adds normal and high priority constants to AlertFeedItem.rb and DB field to associated table
* Adds custom alert feed to Puget Sound that allows manual creation of high priority alerts

### CODE REVIEW

File: app/models/puget_sound_custom_alert_feed.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class PugetSoundCustomAlertFeed < AlertFeed
+  def fetch
+    # no-op
+    super
+  end
+
+  def add_alert_item(title, summary, url)
```
Comment (sayhiben) [inline app/models/puget_sound_custom_alert_feed.rb]: Remember this method the next time we implement an AlertFeed class that needs to manually create alerts; we may benefit from moving this to the base class.

File: app/models/puget_sound_custom_alert_feed.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class PugetSoundCustomAlertFeed < AlertFeed
```
Comment (sayhiben) [inline app/models/puget_sound_custom_alert_feed.rb]: This is good - I wonder if `PugetSoundManualAlertFeed` would be more explicit? Hard to say.

Comment (aaronbrethorst) [inline app/models/puget_sound_custom_alert_feed.rb]: good call.

File: app/models/puget_sound_custom_alert_feed.rb (hunk 1 above)
Comment (aaronbrethorst) [inline app/models/puget_sound_custom_alert_feed.rb]: 👍

**Pull request merged**